src/main/java/com/denizenscript/depenizen/bukkit/properties/breweryx/BreweryPlayerExtensions.java

```diff
@@ -0,0 +1,55 @@
+package com.denizenscript.depenizen.bukkit.properties.breweryx;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.dre.brewery.BPlayer;
+
+public class BreweryPlayerExtensions {
+
+    public static void register() {
+        // <--[tag]
+        // @attribute <BPlayerTag.drunkenness>
+        // @returns ElementTag
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the drunkness of the brewery player.
+        // -->
+        PlayerTag.tagProcessor.registerTag(ElementTag.class, "drunkenness", (attribute, object) -> {
+            BPlayer bPlayer = BPlayer.hasPlayer(object.getPlayerEntity()) ? BPlayer.get(object.getPlayerEntity()) : null;
```

Comment:
If this can null out, that should be explain in the tag meta. If this shouldn't have in practice, shouldn't have all the redundant checks